lib/praxis/docs/generator.rb

```diff
@@ -85,9 +88,9 @@ def collect_reachable_types( data, reachable_types )
         when Hash
           if data.key?(:type) && data[:type].kind_of?(Hash) && ( [:id,:name,:family] - data[:type].keys ).empty?
             type_id = data[:type][:id]
-            unless type_id.nil?
+            unless type_id.nil? || type_id == Praxis::SimpleMediaType.id #SimpleTypes shouldn't be collected
```

Comment:
Because it is not an attributor Type ... it's a simplified type and smells a tiny bit like it.
I went down the route of including the Type and all, but then it needs to implement things like `#example` and such...and it seems way to heavy for what it is...so much so, that I reverted that and went back to this simple route